source/retryable-writes/tests/README.rst

```diff
@@ -296,9 +296,28 @@ supported write operations:
   - ``bulkWrite()`` with ``ordered=true`` (no ``UpdateMany`` or ``DeleteMany``)
   - ``bulkWrite()`` with ``ordered=false`` (no ``UpdateMany`` or ``DeleteMany``)
 
+Prose Tests
+===========
+
+The following tests ensure that retryable writes work properly with replica sets
+and sharded clusters.
+
+#. Test that retryable writes raise an exception when using the MMAPv1 storage engine,
```

Comment:
Suggest rephrasing this for clarity:

```
Test that attempting a write operation with retryable writes enabled raises an exception when using the MMAPv1 storage engine (available on MongoDB <= 4.0). Start by initializing a client with ``retryWrites=True`` and attempt a write operation (e.g. ``insertOne``, ``insertMany``). This should generate an exception with the error code set to 20, and the following error message::

...
```

The current language makes it seem like the MMAPv1 storage engine is only available on 3.6 and 4.0 which is inaccurate. I am also not sure if it makes sense to specify the type of the exception (``IllegalOperation``) as not all drivers have a 1:1 correspondence between exceptions that they raise vs. those that are raised by the server (PyMongo, for instance, raises an `OperationFailure` exception in this case). However, since the pseudocode pertaining to this change does explicitly mention this exception, I would be okay if we also used it here.

Reply:
I have removed the `IllegalOperation` reference and reworded the prose.